UsersController.cs

```diff
@@ -0,0 +1,9 @@
+[HttpPost("delete/{id}")]
```

Comment:
HTTP verb should be `HttpDelete` instead of 'HttpPost'.
`delete` part might not be necessary, http verb already states what action is taken - depends on convention.

```suggestion
[HttpDelete("{id}")]
```